registry/kubevirt/0.0.1/kubevirt.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,252 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
where did this file come from?  I'd expect this to be vendored from the KubeVirt packages once the kubevirt olm manifest generator is merged.